mount_config.go

```diff
@@ -169,6 +169,11 @@ type MountConfig struct {
 	// default name involving the string 'osxfuse' is used.
 	VolumeName string
 
+	// OS X only.
+	//
+	// FuseType choice FUSE impl By OsxFuse/Fuse-T,default is OsxFuse
```

Comment:
This changes behavior, because currently the default is FuseT. Can you change the default to match the current behavior?

Reply:
okay, I will change it